python/python/tests/test_table.py

```diff
@@ -756,9 +756,9 @@ def test_empty_query(db):
     df = table.search().select(["id"]).to_pandas()
     assert len(df) == 10
     df = table.search().select(["id"]).limit(None).to_pandas()
-    assert len(df) == 100
+    assert len(df) == 10
     df = table.search().select(["id"]).limit(-1).to_pandas()
-    assert len(df) == 100
+    assert len(df) == 10
```

Comment:
This is a regression. Passing `None` should mean return all rows, not default to 10. This test should be left as is.